modules/Microsoft.Resources/deploymentScripts/deploy.bicep

```diff
@@ -91,6 +92,8 @@ var storageAccountSettings = !empty(storageAccountResourceId) ? {
   storageAccountName: last(split(storageAccountResourceId, '/'))
 } : {}
 
+var environmentVariablesList = !empty(environmentVariables) ? environmentVariables.secureList : []
```

Comment:
Thought about that, but I kept it consistent with the Key vault implementation. Not opinionated about placing it directly in the resource if you think it makes more sense.

Reply:
Thanks @eriqua that would be great. I updated the module to embed the condition in the resource.

Reply:
I had a look at the KV module. In that case I think it makes sense to use a variable, since we're iterating on it to deploy child secrets. I anyway opened a general issue to align variable usage and guidelines.